docs/data-masking-comparison.md

```diff
@@ -0,0 +1,14 @@
+# Compare the data masking component to the data masking plugin
+
+Percona Server for MySQL 8.0.33 adds the data masking component. Either the component or the plugin extends the server's functionality, but the architecture of a plugin is different than the component. 
+
+The component supports more arguments to the `gen_rnd_email()` function. The main differences between the component and plugin are the following:
```

Comment:
I don't think that `gen_rnd_email` is that important that we want to mention it separately, it could be included among the other differences.

Reply:
Removed.